dot/network/config.go

```diff
@@ -94,6 +95,9 @@ type Config struct {
 	// privateKey the private key for the network p2p identity
 	privateKey crypto.PrivKey
 
+	// private hex encoded Ed125519 key to build p2p identity
+	NodeKey string
```

Comment:
nit comment should start with `// NodeKey ...` from some linters opinionated opinions (maybe just govet?)

```suggestion
	// NodeKey is the private hex encoded Ed125519 key to build the p2p identity.
	NodeKey string
```